### PR TITLE
Add Github issue template

### LINKED ISSUE #13774: Add an Issue Template
Such a template would automatically fill the issue post with a guideline and could help to assure that certain key information are provided. It would be more prominent than the `OpenRA Contributing Guidelines`. 
A drawback could be that it would scare off people for some reason or would be too much effort for its benefits.

Github-Documentation:
https://help.github.com/articles/creating-an-issue-template-for-your-repository/

Such a template can also be added for PRs but I don't see any advantage than e.g. a reminder which issue gets fixed (`Fixes: #...`) or a motivational quote to do reviews 😄. 

<details>
<summary>Example template (not adapted to OpenRA)</summary>

[Source-SublimeTextIssues](https://github.com/SublimeTextIssues/Core/blob/master/ISSUE_TEMPLATE.md)

```markdown
When submitting a feature or enhancement request:

1. Explain briefly what the enhancement is and why you think it would be useful.
2. Provide any other necessary or useful information regarding your issue, such as (code) examples or related links.


When submitting a bug report, please follow the following template:

---

### Summary

...

### Expected behavior

...

### Actual behavior

...

### Steps to reproduce

1. First step
2. Second step
3. Third step

### Environment

* Operating system and version:
  * Windows ...
  * Mac OS ...
  * Linux ...
* Monitor:
  * Resolution ...
  * `dpi_scale` used in ST ...
* Sublime Text:
  * Build 3...
  * ... bit

```
</details>

### PR DESCRIPTION
**Pull request by ltem**
Closes #13774.

This PR adds an issue template to the repo and should be used as a guideline for bug reports and enhancement requests.

If somebody is interested to see the template in action, I created a test-repo [Link](https://github.com/ltem/template_test/issues/new) (Updated on 2018-02-24).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!--
+  This is a guideline, which shall help to write enhancement requests or bug reports.
+  Fill in the placeholders below. Delete any headings and placeholders that you do not use.
+
+  Before you start check if a similar request/bug report already exists in this Github issue tracker and comment there.
+-->
+
+When submitting a feature or enhancement request:
```
Comment (Unit158) [inline .github/ISSUE_TEMPLATE.md]: This should probably be in a comment. (Down to the Issue Summary)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- **Operating System:** [e.g. Windows 10, Mac OS 10.12, Ubuntu 16.04, ...]
+- **.NET / Mono Version:** [e.g. .NET 4.7.1, Mono 4.6.2, ...]
+- **OpenRA Version:** [e.g. release-20171014, playtest-20180208, ...]
+- **Mod:** [e.g. Red Alert, Tiberian Dawn, Dune2000, ...]
+
+#### Additional Information:
+- Steps to reproduce
+<!--
+1. Step
+2. Step
```
Comment (Unit158) [inline .github/ISSUE_TEMPLATE.md]: I'd say these should be outside of a comment (remove the comment tags?)

**Review by Unit158 — CHANGES_REQUESTED** [review]
Looks good, just the way comments are used is not consistent, so that should probably be fixed

**Comment by ltem** [comment]
Updated

**Comment by Unit158** [comment]
:+1:

**Review by Unit158 — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Pull request merged**